### PR TITLE
Add test of an inline resource definition

### PR DESCRIPTION
**Pull request by chrisdunelm**

### CODE REVIEW

File: Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs [hunk 1]
```diff
@@ -85,6 +85,97 @@ public static bool TryParse(string simpleresource, out SimpleResourceName result
...
+
+        /// <summary>
+        /// Parses the given <c>SimpleInlineResource</c> resource name in string form into a new
+        /// <see cref="SimpleInlineResourceName"/> instance.
+        /// </summary>
+        /// <param name="simpleinlineresource">
+        /// The <c>SimpleInlineResource</c> resource name in string form. Must not be <c>null</c>.
+        /// </param>
+        /// <returns>The parsed <see cref="SimpleInlineResourceName"/> if successful.</returns>
+        public static SimpleInlineResourceName Parse(string simpleinlineresource)
```
Comment (jskeet) [inline Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs]: The parameter name doesn't look right here. Where does this come from?

Comment (jskeet) [reply Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs]: (It's nasty in the other one as well - sorry for missing it before.)

Comment (chrisdunelm) [reply Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs]: Yes, you're right, these need improving.
I'd rather fix this in the next PR if that's OK?

Comment (jskeet) [reply Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs]: Fine by me :)

Comment (chrisdunelm) [reply Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNamesResourceNames.cs]: Excellent :)

**Pull request merged**